_posts/2020-02-12-myft-machine-learning.html

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title:  "Traversing the data mines"
+date:   2020-02-12 09:30:30 +0100
+categories: Research
+teaser: Research
+preview-image: /assets/uploads/2019/01/myft_ml_header_graph.png
+excerpt: >
+  Investigating FT user data
+---
+{% include figure.html src="/assets/uploads/2020/02/myft_ml_header_graph.png" width="80%" %}
+
+<p>Interested in the field of Machine Learning and guided by the FT's goal of adding further value through personalisation, we saw an opportunity to have a go at creating our own FT Labs content recommendation engine<sup>tm</sup>. Our early estimates warned this could be quite difficult, and we were right.</p>
+
+<h2>What did we do?</h2>
+
+<p>Data is the cornerstone of any Machine Learning project. If you don't spend the time to gather, prune and enrich your data you will be left with low quality data - Garbage in, garbage out (<a href="https://en.wikipedia.org/wiki/Garbage_in,_garbage_out">https://en.wikipedia.org/wiki/Garbage_in,_garbage_out</a>). Through all of our reading and watching on how to get started with ML projects it was greatly impressed upon us that understanding the data and its context is essential.</p>
+
+<p>The best metrics for evaluating/calculating personalisation would be any data points generated by a user. Therefore we would need to get hold of select user behaviour data sets, not currently open to us. Namely:</p>
+
+<ul>
+<li>Follows/unfollows - user action of choosing to follow an FT topic or the reverse</li>
+<li>Saves/unsaves - user action of choosing to save an article for later or the opposite</li>
+<li>Reads - articles a user has read</li>
+</ul>
+
+<p>Fortunately we were granted the keys to the FT shared large data solution<sup>tm</sup>, unfortunately there are so many datasets contained within it was hard to know where to start.</p>
```

Comment:
TM is capital, can also use HTML entity `&trade;` instead of `<sup>`

Reply:
Done